### PR TITLE
Create Castopod subdomain file

### PR DESCRIPTION
**Pull request by rastacalavera**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]
	
# This commit adds a proxy config for [castopod](https://castopod.org/)

### CODE REVIEW

File: castopod.subdomain.conf.sample [hunk 1]
```diff
@@ -0,0 +1,38 @@
+## Version 2022/09/08
+# make sure that your dns has a cname set for <container_name> and that your <container_name> container is not using a base url
```
Comment (pagdot) [inline castopod.subdomain.conf.sample:2]: would replace <container_name> with castopod

Comment (rastacalavera) [reply castopod.subdomain.conf.sample]: fixed

File: castopod.subdomain.conf.sample [hunk 2]
```diff
@@ -0,0 +1,38 @@
+## Version 2022/09/08
```
[the remark this answers is not in the document]
Comment (rastacalavera) [reply castopod.subdomain.conf.sample]: fixed

**Comment by rastacalavera** [comment]
I think I resolved everything, let me know if more is needed.

**Review by nemchik — APPROVED** [review]

**Pull request merged**